jwst/pipeline/calwebb_spec2.py

```diff
@@ -268,6 +268,10 @@ def process_exposure_product(
         # Apply flux calibration
         result = self.photom(input)
 
+        # Close the input file.  We should really be doing this further up
+        # passing along result all the way down.
+        input.close()
```

Comment:
If so, why not go ahead and do it further up?

Reply:
Also, it might be a good time to replace `input` with a different variable name like `input_model` or something. It's probably not a good idea to keep using `input` since it's also a python built-in function

Reply:
don't forget a change log entry ...

Reply:
@jdavies-st got it, and fair enough!